variables.tf

```diff
@@ -72,6 +72,12 @@ variable "guac_connection_setup_path" {
   default     = "/var/guacamole/dbinit"
 }
 
+variable "operations_instance_counts" {
+  type        = map(number)
+  description = "A map specifying how many instances of each type should be created in the operations subnet (e.g. { \"kali\": 1 })."
```

Comment:
See previous comment about adding a list of supported keys.

Reply:
Added in https://github.com/cisagov/cool-assessment-terraform/pull/16/commits/3e2cf9971b31be012a77dc0d310486418685af3f.